values.yaml

```diff
@@ -698,7 +698,7 @@ jenkins:
         ServiceAccount: jenkins
         Containers:
           Jnlp:
-            Image: jenkinsci/jnlp-slave:3.14-1
+            Image: jenkinsci/jnlp-slave:3.26-1
```

Comment:
@ccojocar what about using the jnlp-slave:3.26-1-alpine image which is smaller?

Reply:
It seems to work :).